app/src/lib/store.ts

```diff
@@ -33,7 +33,7 @@ const persistConfig = {
   key: "root",
   version: 1,
   storage,
-  blacklist: ["api"],
+  blacklist: ["api","subsector"],
```

Comment:
We should probably add some more keys here, as I've noticed it gets quite large after a while... Like e.g. the OpenClimate API one.